src/extract.ts

```diff
@@ -415,44 +496,103 @@ function extractCatch2Result(output: string): BenchmarkResult[] {
     return ret;
 }
 
+function extractRobloxResult(output: string): RobloxBenchmarkResult[] {
+    const lines = output.split(/\r?\n/g);
+    const ret = new Array<RobloxBenchmarkResult>();
+    // Example:
+    //   fib(20) x 11,465 ops/sec ±1.12% (91 runs sampled)(roblox-cli version 123.456)
+    //   createObjectBuffer with 200 comments x 81.61 ops/sec ±1.70% (69 runs sampled)(roblox-cli version 123.456)
```

Comment:
can this be `lua runtime version 123.456`?

Reply:
We could but I still think it should stay roblox specific for now. Maybe accept both formats?